src/main/telemetry/crsf.h

```diff
@@ -19,9 +19,22 @@
 
 #include "common/time.h"
 #include "rx/crsf.h"
+#include "telemetry/msp_shared.h"
+
+typedef enum {
+    CRSF_FRAME_START = 0,
+    CRSF_FRAME_ATTITUDE = CRSF_FRAME_START,
+    CRSF_FRAME_BATTERY_SENSOR,
+    CRSF_FRAME_FLIGHT_MODE,
+    CRSF_FRAME_GPS,
+    CRSF_FRAME_DEVICE_INFO,
+    CRSF_FRAME_MSP_RESPONSE
+} crsfFrameType_e;
```

Comment:
This set of enums is unnecessary and also confusing. I'm not sure why you added them in, they should be removed.

Reply:
I think they were there from before.  I can remove them :+1:

Reply:
Amended